docs/oauth2-examples-google.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Use Google as OAuth 2.0 server
+displayed_sidebar: docsSidebar
+---
+<!--
+Copyright (c) 2005-2024 Broadcom. All Rights Reserved. The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Use Google as OAuth 2.0 server
+
+**Google is not supported**
+
```

Comment:
What does the above sentence actually mean? Does Is it saying, you can use google if you want but if you run into issues, RabbitMQ support is not provided to resolve such issues? If so then perhaps we need to be clearer about this here, I think it is best to be 100% clear about what "support" actually means - thanks